conda/environments/cuml_dev_cuda10.0.yml

```diff
@@ -20,9 +20,9 @@ dependencies:
 - dask=2.8*
 - distributed=2.8*
 - dask-ml
-- dask-cuda=0.11*
-- dask-cudf=0.11*
+- dask-cuda=0.12*
+- dask-cudf=0.12*
 - nccl>=2.4
-- libcumlprims=0.11*
+- libcumlprims=0.12*
```

Comment:
Do we have 0.12 now?

Reply:
yeah, it was addressed a week ago: https://anaconda.org/rapidsai-nightly/libcumlprims

Reply:
yeah, it was addressed a week ago: https://anaconda.org/rapidsai-nightly/libcumlprims